### PR TITLE
Fix routing handling

### LINKED ISSUE #92: Crash when adding a route
Adding/editing a route causes the module to crash. The route is indeed created, but the interface just crashes.

### PR DESCRIPTION
**Pull request by imobachgs**
Fixes #92, Close #93. The problem was introduced in #72.

### CODE REVIEW

**Review by teclator — APPROVED** [review]
LGTM

**Review by dgdavid — APPROVED** [review]
LGTM

File: src/lib/wicked/files.js [hunk 1]
```diff
@@ -332,14 +332,9 @@ class IfrouteParser {
...
             const columns = line.split(/\s/).map(column => column !== "-" ? column : undefined);
-
-            routes.push({
-                destination: columns[0],
-                gateway:     columns[1],
-                netmask:     columns[2],
-                device:      columns[3],
-                options:     columns[4],
-            });
+            let [destination, gateway, netmask, device, ...options] = columns;
```
Comment (dgdavid) [inline src/lib/wicked/files.js:335]: Nice!

File: src/components/RoutingTab.test.js [hunk 2]
```diff
@@ -70,6 +70,22 @@ describe('RoutingTab', () => {
...
+
+        expect(await screen.findByText('192.168.2.1')).toBeInTheDocument();
+        userEvent.click(screen.getByRole('button', { name: 'Actions' }));
+        userEvent.click(screen.getByRole('button', { name: 'Edit' }));
+
+        expect(await screen.findByRole('dialog')).toBeInTheDocument();
+        const dialog = screen.getByRole('dialog');
+        userEvent.type(getByLabelText(dialog, /gateway/i), '{backspace}2');
+        userEvent.click(screen.getByRole('button', { name: 'Change' }));
+        expect(await screen.findByText('192.168.2.2')).toBeInTheDocument();
```
Comment (dgdavid) [inline src/components/RoutingTab.test.js:86]: :ok_hand:

**Review by dgdavid — APPROVED** [review]
Still LGTM, thanks for adding more tests.

**Pull request merged**